bokeh/models/tools.py

```diff
@@ -724,16 +724,33 @@ class HoverTool(Inspection):
         are: 'hex' (to display the color as a hex value), and
         'swatch' to also display a small color swatch.
 
-    Additional format options ``safe`` and `Numbro format codes <http://numbrojs.com/format.html>`_
-    can be included in a post-fix brace block on field names. ::
+    Field names that begin with ``@`` are associated with columns in a
+    ``ColumnDataSource``. For instance the field name ``"@price"`` will
+    display values from the ``"price"`` column whenever a hover is triggered.
+    If the hover is for the 17th glyph, then the hover tooltip will
+    correspondingly display the 17th price value.
 
-        [("total", "@total{$0,0.00}"),
-         ("data", "@data{safe}")]
+    Note that if a column name contains spaces, the it must be supplied by
+    surrounding it in curly braces, e.g. ``@{adjusted close}`` will display
+    values from a column named ``"adjusted close"``.
 
-    Including ``{safe}`` after a field name will override automatic escaping
-    of the tooltip data source. Any HTML tags in the data tags will be rendered
-    as HTML in the resulting HoverTool output. See :ref:`custom_hover_tooltip` for a
-    more detailed example.
+    By default, values for fields (e.g. ``@foo``) are displayed in a basic
+    numeric format. However it is possible to control the formatting of values
+    more precisely. Fields can be modified by appending a format specified to
+    the end in curly braces. Some examples are below.
+
+    .. code-block:: python
+
+        "@foo{0,0.000}"    # formats 10000.1234 as: 10,000.123
+
+        "@foo{(.00)}"      # formats -10000.1234 as: (10000.123)
+
+        "@foo{($ 0.00 a)}" # formats 1230974 as: $ 1.23 m
```

Comment:
```
+Note that format specifications are also compatible with column names that
+have spaces. For example ```@{adjusted close}{($ 0.00 a)}`` applies a format
+to a column named "adjusted close".
```